section3/exercises/arrays.js

```diff
@@ -28,39 +28,41 @@ console.log(animals[0]);
 
 // YOU DO: Write code that will reassign the last item in the animals
 // array to "Gorilla"
-
-
+animals[2] = "Gorilla"
 // YOU DO: Write code that will add a new animal (type of your choice) to position 3.
-
-
+animals[3] = "Penguin"
 // YOU DO: Write code that will log the String "Elephant" in the animals array
-
+animals[2] = "Elephant"
+console.log(animals[2])
 
 //-------------------
 // PART 2: Foods: Array Methods
 //-------------------
 
 // YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+var foods = [ "Tacos", "Chips","Salsa","Guac" ];
```

Comment:
Going forward, pay close attention to your styling convention. Technically, this is valid syntax. But no spaces around the curly brackets, just after the commas `["Tacos", "Chips", "etc.."]` is the generally accepted convention.